### PR TITLE
lohView with gender parameter

### PR DESCRIPTION
**Pull request by jkunisak**
Added gender parameter, few variable name edits.

### CODE REVIEW

File: R/lohView.R [hunk 1]
```diff
@@ -36,10 +38,10 @@
 #' @export
 
 lohView <- function(x=NULL, path=NULL, fileExt=NULL, y=NULL, genome='hg19',
-                    step=500000, window_size=1000000, normal=50,
+                    gender=FALSE, step=500000, window_size=1000000, normal=50,
```
Comment (ahwagner) [inline R/lohView.R]: `gender=FALSE` should be changed to `NULL` for readability. We should change this PR to not calculate X or Y chromosomes unless a vector of length equal to the number of samples is provided. This vector should consist of elements from the set {'M', 'F'}.

File: man/lohView.Rd [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 % Please edit documentation in R/lohView.R
 \name{lohView}
 \alias{lohView}
 \title{plot LOH data}
 \usage{
 lohView(x = NULL, path = NULL, fileExt = NULL, y = NULL,
-  genome = "hg19", step = 5e+05, window_size = 1e+06, normal = 50,
-  gradient_midpoint = 20, gradient_low = "#ffffff",
-  gradient_mid = "#b2b2ff", gradient_high = "#000000", theme_layer = NULL)
+  genome = "hg19", gender = FALSE, step = 5e+05, window_size = 1e+06,
```
Comment (ahwagner) [inline man/lohView.Rd]: `gender=FALSE` should be changed here as well.

File: man/lohView.Rd [hunk 3]
```diff
@@ -27,30 +27,35 @@ within the path specified (required if path argument is specified)}
 \item{genome}{character string specifying the genome assembly from which
 input data is based}
 
+\item{gender}{character string specifying whether loh on the X chromosome
+should be calculated for and plotted}
+
```
Comment (ahwagner) [inline man/lohView.Rd]: This should be changed to reflect the vector of gender values.

**Comment by ahwagner** [comment]
Hey @jkunisak, how is progress on this?

**Comment by jkunisak** [comment]
Hey Alex, 
It is done. I believe Zach is now in the process of accepting all the modifications. As of now, lohView has the tile function but does not have the correct sliding window function. 

Best,
Jason Kunisaki

**Comment by ahwagner** [comment]
Thanks, @jkunisak. @zskidmor, will you be pushing those changes to this PR, or should it be closed?

**Comment by zskidmor** [comment]
Just merged this in, sorry @jkunisak must have missed the git email

**Pull request merged**